src/RegisterElectionPage/Collectors.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import List from "@mui/material/List";
+import ListSubheader from "@mui/material/ListSubheader";
+import ListItem from "@mui/material/ListItem";
+import FormControlLabel from "@mui/material/FormControlLabel";
+import Checkbox from "@mui/material/Checkbox";
+
+const Collectors = () => {
+  const collectors = [
```

Comment:
Is there any reason why the collectors are repeating? Is this like a temporary placeholder for us to add more in the future or something?

Reply:
Ya this was just a place holder. I was just doing it to make sure the sizing of the scrolling box was correct

Reply:
None of this is going to work right and still needs connected to the backend. My goal right now is to finish the UI

Reply:
Gotcha sounds good!